### DISCUSSION TITLE
Pager App?

### DISCUSSION THREAD

**Discussion by militantwalrus** [elkowar/eww#757] [General]
Could this thing be used to create a multi-dimensional (2x2) desktop pager for e.g. openbox?

A number of WMs don't have a maintained pager, and that might be great (I'd have a go at it but I've no clue how).

I might even be persuaded to _sponsor_ such a thing, if it could be done as well as FVWM's pager (but able to apply to other WMs - FWWM's pager is not portable).

**Comment by elkowar** [comment]
What exactly does "pager" mean in this context?

**Reply by militantwalrus** [reply]
A miniaturized representation of the set of virtual desktops. Here's a screenshot of mine on FVWM
![pager_screenshot](https://user-images.githubusercontent.com/1735529/235198060-e1ca52ef-6f28-47b1-bc86-40b52fd58b30.png)
That's 2 rows of 5 desktops (each desk is across 2 monitors)
The gray boxes are currently unfocused windows, the blue one is the browser window (the one I'm typing in now).

The pager lets you click into them to "travel" to a particular desk.

Things like gnome and KDE have often embedded a single row version (which typically don't represent the windows) into their "taskbar".

https://userbase.kde.org/Plasma/Pager might also convey
